### PR TITLE
sends time as seconds

### PR DESCRIPTION
**Pull request by Henry-Tottle**

### CODE REVIEW

**Review by CarlOllington — COMMENTED** [review]
Great work!

File: functions.js [hunk 1]
```diff
@@ -32,17 +32,19 @@ const generateJourneySummaries = (journeys, originCode) => {
             const from = firstStation.name
             const to = lastStation.name
             const line = firstStation.line
-            const stationBreakdown = journey.map(station => station.name)
+            const time = journey.map(station => station.timeToNext).reduce((accumulator, currentValue) => accumulator + currentValue)
```
Comment (matt-ioa) [inline functions.js]: You could totally do this with just the reduce

File: functions.js [hunk 2]
```diff
@@ -32,17 +32,19 @@ const generateJourneySummaries = (journeys, originCode) => {
...
             summaries.push({
-                from: from, to: to, line: line, stations: stationBreakdown
+                from: from, to: to, line: line, time: time, stations: stationBreakdown
             })
         } else {
             const from = lastStation.name
             const to = firstStation.name
             const line = lastStation.line
-            const stationBreakdown = journey.map(station => station.name).reverse()
+            const time = journey.map(station => station.timeToPrev).reduce((accumulator, currentValue) => accumulator + currentValue)
```
Comment (matt-ioa) [inline functions.js]: Same here

File: functions.js [hunk 3]
```diff
@@ -32,17 +32,19 @@ const generateJourneySummaries = (journeys, originCode) => {
...
+            const stationBreakdown = journey.map(station => [station.name, station.timeToPrev]).reverse()
             summaries.push({
-                from: from, to: to, line: line, stations: stationBreakdown
+                from: from, to: to, line: line, time: time, stations: stationBreakdown
             })
```
Comment (matt-ioa) [inline functions.js:66]: Again, a bit of repetition here, consider making a function out of these

File: functions.js [hunk 4]
```diff
@@ -51,20 +53,16 @@ const generateJourneySummaries = (journeys, originCode) => {
 
 const getJourneys = async (req, res) => {
     try {
-        // Get Query Params & Check they're populated. Throw Bad Request.
```
Comment (matt-ioa) [inline functions.js:70]: I actually found these comments quite helpful, why did you remove them?

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Some changes requested

File: functions.js (hunk 3 above)
Comment (Henry-Tottle) [reply functions.js:66]: agreed on not doing this

**Review by matt-ioa — APPROVED** [review]
Great work!
